src/lib/api/supabase/projects/api.ts

```diff
@@ -0,0 +1,89 @@
+import { createServerSupabaseClient } from '@supabase/auth-helpers-nextjs'
+import { DBProject, DBProjectQueryOpts } from 'models/dbProject'
+import { Json } from 'models/json'
+import { NextApiRequest, NextApiResponse } from 'next'
+import { Database } from 'types/database.types'
+import { dbProjects } from '../clients'
+
```

Comment:
all functions here were previously used for sepana, but they've been updated for supabase and the file has been renamed